heltour/tournament/forms.py

```diff
@@ -44,6 +61,14 @@ def __init__(self, *args, **kwargs):
         self.fields['classical_rating'] = forms.IntegerField(required=True, label=_(
             'Your Lichess %s Rating' % rating_type))
 
+        help_text_provisional = 'Applications are only reviewed if you have an established rating. \
```

Comment:
I feel that we should review the text here. I think it can be simplified. 

Team:
> You may apply with a provisional rating, but your application will only be reviewed once your rating is established.

Not-team:
> You may apply with a provisional rating, but your application will only be reviewed once your rating is established unless there are league-specific rules that override this.